### ISSUE TITLE
Would it only work for projects that build with vue-loader ?

### ISSUE THREAD

**Issue by jmaister** [mimani/vue-just-another-dropdown#2]

**Comment by mimani** [comment]
No, there should not be any such dependency, Are you facing any issue using it?

**Comment by jmaister** [comment]
As the package.json file only points to a .vue file, it will be needed something to process it. You can't just load a .vue file in the browser.

**Comment by mimani** [comment]
@jmaister Right. I am not getting time to make relevant changes and test this. If you want you can make the changes, Please feel free to make the changes and test it to make more generic.
